event_routing_backends/processors/xapi/openedx_filters/decorators.py

```diff
@@ -0,0 +1,53 @@
+"""
+Decorators that helps to implement the XAPI filter functionality.
+"""
+import functools
+
+from event_routing_backends.processors.xapi.openedx_filters.filters import XApiBaseFilter
+
+
+def openedx_filter(filter_type):
+    """
+    This decorator allows to implement the  XApiBaseFilter on multiple class methods
+    and intends to modify the returned value from methods like get_actor or get_objects
+    in cases where the standard output doesn't satisfy the implementation requirements.
+
+    Arguments:
+        filter_type: String that defines the filter_type attribute of XApiBaseFilter,
+            this allows to identify the configuration setting.
+        key: String that will be used to extract the value from the returned dictionary
```

Comment:
I am not sure if "key" is a separate argument. it seems like something that is derived from the argument `filter_type`

Reply:
belongs to an old implementation